client/src/App.css

```diff
@@ -128,33 +100,26 @@ nav a.active {
     box-shadow: rgba(0, 0, 0, 0.35) 0px 5px 15px;
 }
 
-
-
 /* buttons */
 
 .login-signin-buttons {
  display: flex;
  justify-content: center;
  gap: 20px;
-
 }
 
-
 button {
     padding: 15px 25px;
     background-color: rgb(219, 104, 104);
     border: none;
-    border-radius: 7px;
-    display: block;
+    border-radius: 7px;   
     width: 110px;
-    margin: 20px auto;
     text-decoration: none;
     color: rgb(251, 249, 249);
```

Comment:
What is the contrast ratio between this text color and the background color? I haven't checked but reading the rgb values my guess is that the contrast is too low to pass Lighthouse.  Do you know?